sylt-parser/src/statement.rs

```diff
@@ -336,15 +336,41 @@ pub fn statement<'t>(ctx: Context<'t>) -> ParseResult<'t, Statement> {
                     format!("{}.sy", name)
                 })
             };
-            let ctx = expect!(ctx, T::Use, "Expected 'use' after path");
-            let (ctx, name) = match ctx.tokens_lookahead::<1>() {
-                [T::Identifier(name)] => (
-                    ctx.skip(1),
-                    Identifier { name: name.clone(), span: ctx.span() },
-                ),
-                _ => raise_syntax_error!(ctx, "Expected identifier"),
+            let mut ctx = expect!(ctx, T::Use, "Expected 'use' after path");
+            let mut imports = Vec::new();
+
+            let paren = matches!(ctx.token(), T::LeftParen);
+            let (ctx_, skip_paren) = if paren {
+                ctx = ctx.skip(1);
+                ctx.push_skip_newlines(true)
+            } else {
+                ctx.push_skip_newlines(false)
+            };
+            ctx = ctx_;
+            loop {
+                match ctx.token() {
+                    T::RightParen | T::Newline => break,
+                    T::Identifier(name) => {
+                        imports.push(Identifier { name: name.clone(), span: ctx.span() });
+                        ctx = ctx.skip(1);
+                        if matches!(ctx.token(), T::Comma | T::RightParen) {
+                            syntax_error!(ctx, "Expected ',' after imports");
+                        }
+                        ctx = ctx.skip_if(T::Comma);
```

Comment:
This seems very wrong, but it works?

Reply:
Took a closer look and it seems to just not do anything.

Reply:
It skips if it's a comma! :D Which is exactly what the code says.

Reply:
I mean the error. Anyway, it should be a quick fix.